src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -4902,6 +4903,36 @@ void ReportZoneMeanAirTemp(EnergyPlusData &state)
     // PURPOSE OF THIS SUBROUTINE:
     // This subroutine updates the report variables for the AirHeatBalance.
 
+    if (state.dataHeatBalAirMgr->CalcExtraReportVarMyOneTimeFlag) {
+        for (auto const *reqVar : state.dataOutputProcessor->reqVars) {
+            if (reqVar->name == "ZONE WETBULB GLOBE TEMPERATURE") {
+                if (reqVar->key.empty()) {
+                    for (int ZoneLoop = 1; ZoneLoop <= state.dataGlobal->NumOfZones; ++ZoneLoop) {
+                        if (state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+                            for (int spaceNum : state.dataHeatBal->Zone(ZoneLoop).spaceIndexes) {
+                                auto &thisSpaceAirRpt = state.dataHeatBal->spaceAirRpt(spaceNum);
+                                thisSpaceAirRpt.ReportWBGT = true;
+                            }
+                        } else {
+                            auto &thisZnAirRpt = state.dataHeatBal->ZnAirRpt(ZoneLoop);
+                            thisZnAirRpt.ReportWBGT = true;
+                        }
+                    }
+                } else {
+                    if (state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+                        int spaceNum = Util::FindItemInList(reqVar->name, state.dataHeatBal->space);
+                        auto &thisSpaceAirRpt = state.dataHeatBal->spaceAirRpt(spaceNum);
+                        thisSpaceAirRpt.ReportWBGT = true;
+                    } else {
+                        int ZoneLoop = Util::FindItemInList(reqVar->name, state.dataHeatBal->Zone);
```

Comment:
Again, this shouldn't be an `else`.

Reply:
indeed

Reply:
Although here there should probably still be a separation based on `state.dataHeatBal->doSpaceHeatBalanceSimulation` T/F values, as when the spaceNum is found, the corresponding zone index containing this space will come from the `zoneNum` field of the space. When there's no space level simulations, a `FindItemInList` will be used to find the zone index